src/plugins/data/public/search/tabify/buckets.test.ts

```diff
@@ -19,6 +19,11 @@
 
 import { TabifyBuckets } from './buckets';
 import { AggGroupNames } from '../aggs';
+import moment from 'moment';
+
+interface Bucket {
```

Comment:
defining an interface in tests ?

Reply:
The previous definition is `any` and I just didn't want to continue using `any` below. I could have also inlined this type, but since I used it in multiple places just defined it as an interface on top.